types/next-auth/client.d.ts

```diff
@@ -28,6 +28,19 @@ interface SetOptionsParams {
     keepAlive?: number;
 }
 
+interface SignInObjectReturn {
+    error: string | undefined;
+    status: number;
+    ok: boolean;
+    url: string | null;
+}
+
+type SignInDataParams = GenericObject & {
+  callbackUrl?: string;
+};
+
+type SignInAuthorizationParams = string | string[][] | GenericObject | URLSearchParams;
```

Comment:
Same, couldn't this just be inline?

Reply:
I only created this type to avoid code repeat. But I could use inline

Reply:
@balazsorban44 done